device-types/Dell/PowerEdge-MX7000.yaml

```diff
@@ -2,22 +2,62 @@
 manufacturer: Dell
 model: PowerEdge MX7000
 slug: dell-poweredge-mx7000
-u_height: 7
+part_number: '5112814'
+u_height: 7.0
 is_full_depth: true
 subdevice_role: parent
+airflow: front-to-rear
+weight: 105.0
+weight_unit: lb
+comments: '[Product Page](https://www.dell.com/en-us/shop/ipovw/poweredge-mx7000)'
 power-ports:
-  - name: PSU.Slot.1
+  - name: PSU.IN.1
     type: iec-60320-c22
-  - name: PSU.Slot.2
+    max_draw: 3000
+    label: 1
+  - name: PSU.IN.2
     type: iec-60320-c22
-  - name: PSU.Slot.3
+    max_draw: 3000
+    label: 2
+  - name: PSU.IN.3
     type: iec-60320-c22
-  - name: PSU.Slot.4
+    max_draw: 3000
+    label: 3
+  - name: PSU.IN.4
     type: iec-60320-c22
-  - name: PSU.Slot.5
+    max_draw: 3000
+    label: 4
+  - name: PSU.IN.5
     type: iec-60320-c22
-  - name: PSU.Slot.6
+    max_draw: 3000
+    label: 5
+  - name: PSU.IN.6
     type: iec-60320-c22
+    max_draw: 3000
+    label: 6
+module-bays:
+  - name: IOM-A1
+    position: A1
+  - name: IOM-A2
+    position: A2
+  - name: IOM-B1
+    position: B1
+  - name: IOM-B2
+    position: B2
+  - name: IOM-C1
+    position: C1
+  - name: IOM-C2
+    position: C2
+  - name: MM-1
+    position: MM-1
+  - name: MM-2
+    position: MM-2
+  - name: PSU.Slot.1
```

Comment:
What would be cool, of course, is if you could also add the most common PSUs as modules to `module-types/Dell/`.

Reply:
I've added 4 PSU's.

Currently we have only `H7TFG` so the pick was mostly out of number of results/availability. The minimum that needs to be present is one AC (e.g. H7TFG) and one DC (e.g. 0155M8).